src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -80,7 +81,7 @@ def __call__(self, path, **params):
                 data = cherrypy.request.body.read()
 
             return rgw_client.proxy(method, path, params, data)
-        except RequestException as e:
+        except (NoCredentialsException, RequestException) as e:
```

Comment:
@p-na `NoCredentialsException` is only used in the rgw controller. Would it make sense to make `NoCredentialsException` inherent from `RequestException`?

Reply:
Makes sense